cfpb/cfgov-refresh
# Ask CFPB: Add responsive css for YouTube videos

Add [Wagtail-recommended](https://github.com/wagtail/wagtail/issues/1801) css for responsive YouTube embeds

## Additions

- CSS for Ask answer YouTube embeds

## Testing

1. Check out branch & run `gulp styles`
2. Add a YouTube video to an English and Spanish answer in admin and publish
3. Navigate to answer pages and verify that videos appear correctly (without a large amount of blank space after them) and resize across screen sizes

## Screenshots
<img width="921" alt="screen shot 2017-06-19 at 8 08 47 pm" src="https://user-images.githubusercontent.com/778171/27314935-9975041a-552b-11e7-9a42-e8ca6267a1d3.png">
<img width="920" alt="screen shot 2017-06-19 at 8 06 41 pm" src="https://user-images.githubusercontent.com/778171/27314934-9971cf52-552b-11e7-962e-2f0605a6f242.png">


## Checklist
